docs/api-reference/next/future/image.md

```diff
@@ -175,6 +177,12 @@ The `height` property represents the _rendered_ height in pixels, so it will aff
 
 Required, except for [statically imported images](/docs/basic-features/image-optimization.md#local-images) or images with the [`fill` property](#fill).
 
+### alt
+
+The `alt` property is used to describe the image for screen readers and search engines. It is also the fallback text if images have been disabled or an error occurs while loading the image.
+
+It should contain replacement text that could be used instead of the image. It is not meant to supplement the image and should not repeat information that is already provided in the captions above or below the image.
```

Comment:
```suggestion
The `alt` property is used to describe the image for screen readers and search engines. It is also the fallback text if images have been disabled or an error occurs while loading the image.

It should contain replacement text that could be used instead of the image. It is not meant to supplement the image and should not repeat information that is already provided in the captions above or below the image.

If the image is decorative, the `alt` property should be an empty string (`alt=""`). See [A purely decorative image that doesn't add any information - HTML5 Spec](https://html.spec.whatwg.org/multipage/images.html#a-purely-decorative-image-that-doesn't-add-any-information)
```